packages/component-submission/server/aggregates/Submission.js

```diff
@@ -0,0 +1,33 @@
+class Submission {
+  constructor({ models: { Manuscript, File }, services: { Storage } }) {
+    this.ManuscriptModel = Manuscript
+    this.FileModel = File
+    this.Storage = Storage
+  }
+
+  async initialize(manuscriptId, userId) {
+    this.manuscript = await this.ManuscriptModel.find(manuscriptId, userId)
+    this.files = await this.FileModel.findByManuscriptId(manuscriptId)
+    console.log(this.manuscript, this.files)
+    return this
+  }
+
+  getFilesWithDownloadLink() {
```

Comment:
Should this be a public method ? Might be best to keep it private?

Reply:
This is used as a private method, however I didn't think there was any way to set real private methods in node currently (???). I'd be happy to use a naming convention to make them identifiable, something like `_getFilesWithDownloadLink()`?

Reply:
one way is to use the ES6 Symbol:

top of the file:
```
const _getFilesWithDownloadLink = Symbol('getFilesWithDownloadLink')
```

Inside the class:
```
[_getFilesWithDownloadLink]() {
//...
}
```